navigation/index.tsx

```diff
@@ -20,11 +23,16 @@ export type RootStackParamList = {
   Settings: undefined
   Signaling: undefined
   Search: undefined
+  NoFriends: undefined
 }
 
 export default function AppNavigator() {
   const Stack = createNativeStackNavigator<RootStackParamList>()
   const { isAuthenticated, isLoading } = useAuth()
+  const hasFriends = useMemo(() => {
+    const friends = [...availableFriends, ...offlineFriends]
+    return friends.length !== 0
+  }, [])
```

Comment:
we won't be checking for friends in this level. once we have the BE for that, we should do the query inside `Home.tsx`